server/src/utils/getConfig.test.ts

```diff
@@ -138,13 +141,16 @@ describe('config', () => {
       .spyOn(getDefaultConfig, 'getDefaultConfig')
       .mockImplementation((): any => mockDefaultConfig);
 
-    const config = getConfig.getServerConfig();
+    const serverConfig = getConfig.getServerConfig();
+    serverConfig.communicationServicesConnectionString = 'endpoint=test_endpoint_value;accesskey=secret';
+    const clientConfig = getConfig.getClientConfig(serverConfig);
+    const options: CustomSurveyOptions = clientConfig?.postCall?.survey.options as CustomSurveyOptions;
     expect(getDefaultConfigSpy).toHaveBeenCalled();
-    expect(config.postCall).toBeDefined();
-    expect(config.postCall?.survey).toBeDefined();
-    expect(config.postCall?.survey.type).toBe(mockDefaultConfig.postCall.survey.type);
-    expect(config.postCall?.survey.options).toBeDefined();
-    expect(config.postCall?.survey.options?.surveyUrl).toBe(mockDefaultConfig.postCall.survey.options.surveyUrl);
+    expect(clientConfig.postCall).toBeDefined();
+    expect(clientConfig.postCall?.survey).toBeDefined();
+    expect(clientConfig.postCall?.survey.type).toBe(mockDefaultConfig.postCall.survey.type);
+    expect(clientConfig.postCall?.survey.options).toBeDefined();
+    expect(options.surveyUrl).toBe(mockDefaultConfig.postCall.survey.options.surveyUrl);
   });
 
   test('getServerConfig returns empty postcall object when postCallSurveyType is invalid', () => {
```

Comment:
Nit: I think we should change the test name to be more accurate. `getServerConfig returns undefined when postCallSurveyType is invalid`

Reply:
Fixed this.